app/views/catalog/_show_default.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class='row'>
   <div class='col-md-12'>
-    <ul class='nav nav-tabs nav-fill' role='tablist' aria-label='<%= t('arclight.views.show.tablist_nav') %>'>
```

Comment:
Why have we changed this markup? This seems to be different than the other design tickets?

Reply:
Arg, fixed. :-)